src/components/screenings/ScreeningsListSeance.vue

```diff
@@ -0,0 +1,32 @@
+<script>
+import { defineComponent } from 'vue';
+
+import BaseButton from '@/components/base/BaseButton.vue';
+
+export default defineComponent({
+  components: {
+    BaseButton,
+  },
+  props: {
+    seanceData: {
+      type: Object,
+      required: true,
+    },
+  },
+  computed: {
+    seanceTime() {
+      const datetime = this.seanceData.datetime;
+      const timePoint = datetime.indexOf('T') + 1;
+      return datetime.slice(timePoint).slice(0, 5);
```

Comment:
Nice alternative that's supported by browsers would be to use `toLocaleTimeString`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString

```suggestion
      const date = new Date(this.seanceData.datetime));
      return date.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' }));
```